packages/ui/src/signals/labelClip.ts

```diff
@@ -0,0 +1,11 @@
+import {signal} from '@preact/signals';
+
+interface LabelClipDrag {
+  left: number | null;
+  dragging: boolean;
+}
```

Comment:
I meant to replace the entire interface with `number | null`
Now you have both the nullable `left` and `dragging` carrying duplicated information for no reason.

To be clear, this is what I mean:
```ts
type LabelClipDrag = number | null;

export const labelClipDraggingSignal = signal<LabelClipDrag>(null);
```

Reply:
Changed the type to `<number | null>` for label drag signal